tests/api/runtimes/test_application.py

```diff
@@ -73,7 +73,7 @@ def test_create_function_validate_min_nuclio_version(
             self._generate_runtime(self.runtime_kind)
         assert (
             str(exc.value)
-            == "'ApplicationRuntime.__init__' function requires Nuclio v1.13.1 or higher"
+            == "'Application Runtime' function requires Nuclio v1.13.1 or higher"
```

Comment:
it says `'Application Runtime' function` ... should be just `'Application Runtime'`

Reply:
This is what @jillnogold suggested. 
Please see the conversation here - https://iguazio.atlassian.net/browse/ML-6539